lms/djangoapps/verify_student/views.py

```diff
@@ -971,6 +971,13 @@ def _decode_image_data(self, request, face_data, photo_id_data=None):
             tuple of (str, str, HttpResponse)
 
         """
+        for image_data in [face_data, photo_id_data]:
+            # Validate that the media type is .png, .jpg, or .jpeg
+            if image_data and not image_data.startswith(('data:image/png', 'data:image/jpeg')):
```

Comment:
We should allow more image formats I think. Look at the list at https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types. You probably want to access all types with `image/*`